### PR TITLE
Include description of otelcol Helm chart params

### PR DESCRIPTION
**Pull request by pmm-sumo**
###### Description

Adds docs for OpenTelemetry Collector-related params

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Review by frankreno — APPROVED** [review]
LGTM, thanks @pmm-sumo !

File: deploy/helm/sumologic/README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Parameter | Description | Default
 `sumologic.setupEnabled` | If enabled, a pre-install hook will create Collector and Sources in Sumo Logic. | `true`
 `sumologic.logs.enabled` | Set the enabled flag to false for disabling logs ingestion altogether. | `true`
 `sumologic.metrics.enabled` | Set the enabled flag to false for disabling metrics ingestion altogether. | `true`
+`sumologic.traces.enabled` | Set the enabled flag to true to enable tracing ingestion. | `false`
```
Comment (pmm-sumo) [inline deploy/helm/sumologic/README.md:20]: Perhaps we should add a note like "Tracing must be enabled for the account first. Please contact your Sumo representative for activation details" - is it a fit place @frankreno ? We don't have really good handling of the error when source creation fails due to tracing not being enabled (going to wrap Terraform exception but that's all that can be done easily without more significant changes). So maybe at least such note could help?

Comment (perk-sumo) [reply deploy/helm/sumologic/README.md:20]: Yeah, that's a good idea! Please add it here.

**Review by perk-sumo — APPROVED** [review]
Please resolve conflicts and add info about requirement for tracing being enabled for the account. 
Other than that I'm 👍

**Comment by pmm-sumo** [comment]
@perk-sumo resolved the conflicts and added a note

**Pull request merged**